web/src/pages/dashboard/index.page.jsx

```diff
@@ -1,11 +1,12 @@
+import Template from "@/components/templates/Template";
+import Link from 'next/link';
 
-const Dashboard = () => {
-
-    return (
-        <section>
-            Test
-        </section>
-    )
+const Layout = ({ children }) => {
```

Comment:
- Should be Dashboard, dont rename.
- Link and children is unused, remove

```suggestion
const Dashboard= () => {
```